app/grandchallenge/documentation/views.py

```diff
@@ -0,0 +1,41 @@
+from django.views.generic import DetailView, ListView
+
+from grandchallenge.core.templatetags.bleach import clean
+from grandchallenge.documentation.models import DocPage
+
+
+class DocPageList(ListView):
+    model = DocPage
+
+
+class DocPageDetail(DetailView):
+    model = DocPage
+
+    def get_context_object_name(self, obj):
+        return "currentdocpage"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        firstdocpage = DocPage.objects.first()
+        top_level_pages = (
+            DocPage.objects.filter(parent__isnull=True)
+            .prefetch_related("children__children")
+            .order_by("order")
+            .all()
+        )
+
+        context.update(
+            {
+                "firstdocpage": firstdocpage,
+                "top_level_pages": top_level_pages,
+                "cleaned_content": clean(self.object.content),
+            }
+        )
+
+        return context
+
+
+class DocumentationHome(DocPageDetail):
+    def get_object(self, queryset=None):
+        page = DocPage.objects.first()
+        return page

```

Comment:
This will result in an error when there are no pages yet. We could add an error page as it is done for pages, or instead (since this is just an issue at the very beginning and never again after that), just create a default first page, like so: 
 ```
  if not page:
            page = DocPage.objects.create(title='main')
```

Reply:
Django has something built-in just for that :-) https://docs.djangoproject.com/en/3.2/topics/http/shortcuts/#get-object-or-404

This PR LGTM, only thing I'm wondering is if `display_title` is actually used? This is an artefact from the ancient `Pages` model that uses `title` and `display_title` rather than `slug` and `title`.

Reply:
True, it's not necessary. I thought that the title would otherwise not allow special characters and icons, but it does I see now. ;-)